drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -14,6 +14,14 @@ def hmis_user
     Hmis::Hud::User.from_user(current_user)
   end
 
+  def impersonating?
+    current_user != true_user
+  end
+  
+  def true_hmis_user
+    impersonating? ? Hmis::Hud::User.from_user(true_user) : hmis_user
+  end
```

Comment:
I'm confused by this one. We already have references to `true_hmis_user` which I thought was defined by the 
[pretender](https://github.com/ankane/pretender) gem as the application user id (of the user who is impersonating someone else). See other places we refer to this like [in the base controller](https://github.com/greenriver/hmis-warehouse/blob/997ee79b0c3659f5c503239009e3e71ef6887a5b/drivers/hmis/app/controllers/hmis/base_controller.rb#L74-L76). Now here we are defining the same variable for use in queries/mutations, but this time it's a HUD `User` instead of an application user (`users`).

Am I grasping this? Maybe let's do a huddle on this.